stream_31_10_2019/src/Main.java

```diff
@@ -0,0 +1,180 @@
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.*;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Person p1 = new Person("Ivan", 23, new Address("Mira", 10));
+        Person p2 = new Person("Mariya", 10, new Address("Pushkina", 29));
+        Person p3 = new Person("Moshe", 120, new Address("Tov", 7));
+        Person p4 = new Person("Vova", 16, new Address("Pobedu", 4));
+
+        List<Person> persons = new ArrayList<>();
+        persons.add(p1);
+        persons.add(p2);
+        persons.add(p3);
+        persons.add(p4);
```

Comment:
Can be easily replaced with `List<Person> persons = Arrays.asList(p1, p2, p3, p4);`. The resulting list is immutable though, and you may not be able to add new elements to it if needed.